stream_alert/athena_partition_refresh/main.py

```diff
@@ -450,13 +475,18 @@ def unique_s3_buckets_and_keys(self):
 
         for message in self.received_messages:
             if 'Body' not in message:
-                LOGGER.error('Missing \'Body\' key, trying next SQS message')
+                LOGGER.error('Missing \'Body\' key in SQS message, skipping')
                 continue
 
             loaded_message = json.loads(message['Body'])
 
+            if loaded_message.get('Event') == 'S3:TestEvent':
+                LOGGER.debug('Skipping S3:TestEvent')
```

Comment:
Is this something that will happen in production? Curious why this code exists here. If it should remain please add a comment.